Formula/sass.rb

```diff
@@ -0,0 +1,28 @@
+require "language/node"
+
+class Sass < Formula
+  desc "Pure JavaScript implementation of SASS"
```

Comment:
Having `sass` describe itself as an “implementation of SASS” is not great. Can you think of something better?